packages/cli/templates/app/app/pages/index.tsx

```diff
@@ -8,9 +8,34 @@ const Home = () => (
     </Head>
 
     <main>
+      <div className="logo">
+        <img src="/logo.png" alt="blitz icon" height="72" width="72" />
+      </div>
       <h1 className="title" style={{marginBottom: 24}}>
         Welcome to Blitz.js
       </h1>
+      <p>
+        Get started by editing this page.{' '}
```

Comment:
The code part is in the new line so this space is not required

Reply:
fixed in my branch